core/integration/directory_test.go

```diff
@@ -408,9 +409,12 @@ func TestDirectoryWithTimestamps(t *testing.T) {
 			WithExec([]string{"sh", "-c", "ls -al /dir && ls -al /dir/sub-dir"}).
 			Stdout(ctx)
 		require.NoError(t, err)
-		require.Contains(t, ls, "-rw-r--r--    1 root     root            12 Oct 26  1985 some-file")
-		require.Contains(t, ls, "drwxr-xr-x    2 root     root          4096 Oct 26  1985 sub-dir")
-		require.Contains(t, ls, "-rw-r--r--    1 root     root            11 Oct 26  1985 sub-file")
+		require.Regexp(t, regexp.MustCompile(`-rw-r--r--\s+1 root\s+root\s+\d+ Oct 26  1985 some-file`), ls)
+		require.Regexp(t, regexp.MustCompile(`drwxr-xr-x\s+2 root\s+root\s+\d+ Oct 26  1985 sub-dir`), ls)
+		require.Regexp(t, regexp.MustCompile(`-rw-r--r--\s+1 root\s+root\s+\d+ Oct 26  1985 sub-file`), ls)
+		// require.Contains(t, ls, "-rw-r--r--    1 root     root            12 Oct 26  1985 some-file")
+		// require.Contains(t, ls, "drwxr-xr-x    2 root     root          4096 Oct 26  1985 sub-dir")
+		// require.Contains(t, ls, "-rw-r--r--    1 root     root            11 Oct 26  1985 sub-file")
```

Comment:
```suggestion
```

Reply:
@vito Yeah! Broke on `xfs`, iirc. And whatever filesystem the EKS AMIs use.